daemon/execdriver/native/driver.go

```diff
@@ -355,6 +355,23 @@ func (d *driver) Stats(id string) (*execdriver.ResourceStats, error) {
 	}, nil
 }
 
+func (d *driver) Set(c *execdriver.Command) error {
+	cont := d.activeContainers[c.ID]
```

Comment:
ok, not 100% sure, but I think the panic issue I'm seeing is because you're checking to see if the container is 'active' instead of "running".